Fitz/Features/AccountsRework/AccountsMapper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Fitz.Features.AccountsRework.Create.Discord;
```

Comment:
I put the mapper in top of the noun folder because I don't see the benefit keeping it in the same folder as the adapter (controller). Currently it's only context is for mapping CreateAccountDto to CreateAccountCommand. 

Putting a DeactivateAccountDto -> DeactivateAccountCommand method inside this file in the future seems reasonable to me. I can also see the benefit placing it inside of the action folder itself. In this case, AccountsRework/Create/Discord/AccountsMapper.cs

I would also agree naming here isn't the best.